LnkParse3/lnk_file.py

```diff
@@ -22,13 +22,14 @@
 
 
 class LnkFile:
-    def __init__(self, fhandle=None, indata=None, cp=None):
+    def __init__(self, fhandle=None, indata=None, cp=None, terminal=True):
```

Comment:
The only thing I am wondering about is the parameter's name. `terminal` sounds too vague to me. What about `allow_extra_terminal_data`? Or something similar.